File: week1/day3/calculator.py
num1 = int(input('Enter your first number: '))
operand = input("Enter an operand: ")
num2 = int(input('Enter your second number: '))

def add(num1, num2):
    return print(num1 + num2)
def multiply(num1, num2):
    return print(num1 * num2)
def division(num1, num2):
    return print(num1 // num2)
def subtract(num1, num2):
    return print(num1 - num2)
if (operand == "+"):
    add(num1,num2)
elif (operand == "*"):
    multiply(num1,num2)
elif (operand == "//"):
    division(num1,num2)
elif (operand == "-"):
    subtract(num1,num2)
else:
    print(" ")


    

File: week4/day1/calculator.js
function calculate(num1,operand,num2){
    if (operand == "multiply") {
        console.log(num1*num2)
    }
    else if (operand == "add") {
        console.log(num1+num2)
    }
    else if (operand == "subtract") {
        console.log(num1-num2)
    }
    else if (operand == "divide") {
        console.log(num1/num2)
    }
}

calculate(10,"subtract",4)

